.flowconfig

```diff
@@ -0,0 +1,12 @@
+[ignore]
+.*/node_modules/cypress/*
+.*/node_modules/config-chain/*
+[include]
+
+[libs]
+
+[lints]
+
+[options]
+module.file_ext=.scss
```

Comment:
Unfortunately, this line overwrites the whole config of flow. It should be something like this:
```
module.use_strict=true
module.file_ext=.css
module.file_ext=.scss
module.file_ext=.js
module.file_ext=.jsx
module.file_ext=.json
```

Reply:
Thanks!